src/css/global.css

```diff
@@ -1,5 +1,13 @@
-.btn {
-  @apply text-base py-8 px-16 text-c100 font-extrabold bg-c300;
+.btn-lg {
+  @apply leading-none text-base text-c100 font-extrabold bg-c300 py-8 px-16;
+}
+
+.btn-md {
+  @apply leading-none text-base text-c100 font-extrabold bg-c300 py-7 px-16;
+}
+
+.btn-sm {
+  @apply leading-none text-base text-c100 font-extrabold bg-c300 py-5 px-16;
```

Comment:
The only difference between those three selectors is "py", so don't repeat yourself and make a separate selector to group these repetitive utility classes.
Also can you please put the ticket link in the PR description

Reply:
https://app.clickup.com/t/4rj26g